bin/setupReplication.js

```diff
@@ -100,12 +117,24 @@ class _SetupReplication {
         this._sourceBucket = sourceBucket;
         this._targetBucket = targetBucket;
         // TODO: support bootstrap list failover
-        const destEndpoint = destination.bootstrapList[0].servers[0];
+        const destEndpoint = '217.182.11.96:8000';
         const verifySourceProfile = sourceProfile === undefined ?
             'default' : sourceProfile;
         const verifyTargetProfile = targetProfile === undefined ?
             'default' : targetProfile;
 
+	console.log('dest endpoint: ', destination);
+
+	console.log('sourceBucket:', this._sourceBucket);
+	console.log('targetBucket:', this._targetBucket);
+	console.log(source.s3);
+	console.log('source.auth.vault:', source.auth.vault);
+	console.log('destEndpoint', destEndpoint);
+	console.log('verifySource:', verifySourceProfile);
+	console.log('verifyTarget:', verifyTargetProfile);
+	source.auth.vault.host = 'localhost';
+	source.s3.host = 'localhost';
```

Comment:
These were set as the server name, i.e. `philz-07`. I manually changed to `localhost` for testing